package.json

```diff
@@ -36,7 +37,17 @@
     "not IE 11",
     "not op_mini all"
   ],
-  "exports": "./dist/index.js",
+  "exports": {
+    ".": {
+      "types":"./src/index.d.ts",
+      "import": "./dist/index.js"
+    },
+    "./debug": {
+      "types":"./src/debug.d.ts",
+      "import": "./dist/debug.js"
+    },
+    "./package.json": "./package.json"
```

Comment:
Curious why this is needed? Do people typically need to import `package.json` for some reason?

Reply:
not typically people, but machines do. and without this listing, node fails to tell the bundled about the export map for example. 

here's a few examples: 

svelte: https://github.com/sveltejs/svelte/blob/master/package.json#L24
polka: https://github.com/lukeed/polka/blob/next/packages/polka/package.json#L15
vite: https://github.com/vitejs/vite/blob/main/packages/vite/package.json#L31

but think maybe @lukeed has a better answer for you.

Reply:
it's just a good idea to have in general. things like rollup plugins may try to access a `package.json` file directly, so this just wards off the inevitable request to expose it